joomla/joomla-cms
# [4.0] Respect stage ordering

Pull Request for Issue #21537 .

### Summary of Changes
Ordering of stages are respected now


### Testing Instructions
Reorder stages and check in articles filter (backend) or transition edit view if the ordering of the stages are correct.
